### PR TITLE
Edited get_involved.html

### PR DESCRIPTION
**Pull request by tedmcox**
Updated page to reflect that 2015 CFP and exhibitor info are forthcoming.

Minor punctuation change on Stay in Touch sidebar.

### CODE REVIEW

**Comment by ex-nerd** [comment]
Waiting to hear back from another couple organizers about the changes to the sponsorship stuff.  I'm not sure that info will change.

File: get_involved.html [hunk 1]
```diff
@@ -11,23 +11,16 @@
...
-            We are looking for anyone interested in running a session dealing
-            with any topic related to Free/Libre and Open Source software,
-            hardware, or technology in general. If you would like to host a
-            session please {{ site.custom.a.survey.cpf }}.
-        </p>
-        <p>
-            Update:  The deadline for submissions has been extended until
-            August 3rd.
+            The 2015 Call for Participation will open soon. Check here often! We will look for anyone interested in running a session dealing
+            with topics related to Free/Libre and Open Source software,
```
Comment (edunham) [inline get_involved.html]: Thank you for updating this! One little detail for future PRs is that it's easier to proofread changes in the github UI when lines are wrapped to 80 characters.

**Comment by ex-nerd** [comment]
Weird, thought I replied about this earlier..  The sponsorship info won't be changing for 2015, so we don't need any changes applied to that particular section.  Happy to accept this after you resubmit with that section unmodified (and ideally the line fixed wrapping as @edunham suggested).

**Comment by tedmcox** [comment]
Cool, I'll submit a new pull request.

Do either of you use vim for text editing? Whenever I hit Enter to shorten
a line and then Tab to indent, the indent doesn't line up with the
preceding line. Google hasn't pointed me towards a good solution.

**Comment by edunham** [comment]
te

**Comment by edunham** [comment]
regarding vim, I have the following in my `~/.vimrc`: 
```
set expandtab 
set tabstop=4
set shiftwidth=4
set tw=78
set cc=80
```
Hard tabs will display at various widths depending on your terminal settings, and the `expandtab` and `tabstop` settings combine to insert 4 spaces whenever you hit the tab key. `tw` is text width, the number of columns past which your line will automatically wrap. 

If you have a block of text (with a blank line above and below, to define it as a paragraph), the `gqap` gesture will re-wrap the entire paragraph so each line breaks at the last space before the 78th column. Another useful gesture when doing literary editing in Vim is `J`, which joins the a line to its successor (making one long line), if you were typing and it wrapped and you're stuck with a short line in an awkward spot.

the `cc` part sets a color column at the 80th character, which makes it easy to tell when your lines are too long.

**Comment by tedmcox** [comment]
Thanks, @edunham. Those additions to my .vimrc file were perfect.

The line wrapping has been fixed, the sponsorship section has been reverted, and I've amended the push.

**Comment by ex-nerd** [comment]
And aside from the bit of trailing whitespace, this looks great.

**Pull request merged**